pkg/config/config.go

```diff
@@ -255,6 +270,23 @@ func (lci *LocalConfigInfo) DeleteConfiguration(parameter string) error {
 
 }
 
+// DeleteUrl is used to delete config from local odo config
+func (lci *LocalConfigInfo) DeleteUrl(parameter string) error {
+	for elem, i := range *lci.componentSettings.Url {
+		if i.Name == parameter {
+			po := remove(*lci.componentSettings.Url, elem)
+			fmt.Println(po)
+			lci.componentSettings.Url = &po
+		}
+	}
+	return lci.writeToFile()
+
+}
+
+func remove(s []ConfigUrl, i int) []ConfigUrl {
```

Comment:
may be tie this to an alias of []ConfigUrl ? instead of taking it as a parameter and move the lopping logic from DeleteUrl to here?

Reply:
I didn't understand what you said here :cry:

Reply:
removed `remove` function